tests/std/tests/P0226R1_math_special_functions/test.cpp

```diff
@@ -0,0 +1,495 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cmath>
+#include <limits>
+
+using namespace std;
+
+template <typename Float>
+void assert_close(const Float f, const Float g) {
+    assert(abs(f - g) < 0.001);

```

Comment:
This makes me extremely uncomfortable. My understanding is that these functions should be _way_ closer than $\epsilon \times 10^{12}$...; I'd prefer to see something more similar to $\operatorname{assert}\left(\left|f - g\right| < N \epsilon \times \left|f\right|\right)$